client/src/bundles/bundle.js

```diff
@@ -0,0 +1,3 @@
+require('./UserForms.js');
+require('./FieldEditor.js');
+require('./Recipient.js');
```

Comment:
These paths could probably be done by `import 'bundles/UserForms.js';` etc
`bundles` at the start because webpack-config sets `client/src` as a lookup path for modules, which means later on you don't need to have `../../../grandparent` relative paths.

I also prefer `import` over `require()`

P.S. I think one of these files was meant to be for the frontend only?